idaes/core/util/model_diagnostics.py

```diff
@@ -1450,6 +1450,10 @@ def __init__(self, model: _BlockData, **kwargs):
             self._model, scaled=False, equality_constraints_only=True
         )
 
+        # Get list of equality constraint and variable names
+        self.eq_con_list = self.nlp.get_pyomo_equality_constraints()
```

Comment:
I think you should make these private. There is no reason the user should be changing these.

Reply:
I don't think they should be changing them, but they might have the desire to look at them. We can still make them semiprivate, though.